api/desecapi/pdns.py

```diff
@@ -83,9 +96,12 @@ def create_zone(domain, nameservers):
     salt = '%016x' % random.randrange(16**16)
     payload = {'name': name, 'kind': 'MASTER', 'dnssec': True,
                'nsec3param': '1 0 127 %s' % salt, 'nameservers': nameservers}
-    _pdns_post('/zones', payload)
+    _pdns_post(NSLORD, '/zones', payload)
+
+    payload = {'name': name, 'kind': 'SLAVE', 'masters': [socket.gethostbyname('nslord')]}
```

Comment:
Somehow my gut feeling is that it's cool to use environment variables for configuration in our `settings` module, but otherwise try to avoid using it generally in the code. I thought about adding a setting for that, but that'd be somehow redundant. I then realized that the nslord IP is cached locally anyways (because of the API calls on nslord), so it would be only a very small hit to do it this way. (Even better would be if pdns accepted hostnames in the `masters` field.)

Reply:
okay, fine with me